app/src/types/IdpObjects.ts

```diff
@@ -0,0 +1,6 @@
+export type IdpObjects = {
```

Comment:
How do you feel about `IdpAttributes`? As these are pre filled idp information, and data coming from the SSO attributes object.